Jenkinsfile

```diff
@@ -0,0 +1,15 @@
+currentBuild.description = 'Cause: version bump'
```

Comment:
This file is not needed. We build with docker hub scripts. There is no need for local builds.